src/reducerMaps/restaurants.js

```diff
@@ -152,8 +148,24 @@ export default new Map([
           }
         }
       }
-    })
-  ],
+    };
+
+    if (!state.items.entities.votes || 
```

Comment:
Now that we're checking the index of the result list rather than the existence of entity IDs, I'm not sure if you need to check for vote entities here anymore. Regardless of whether the entities exist, what matters is whether the ID is within the result list or not.